pool/example_pool_test.go

```diff
@@ -1,54 +0,0 @@
-package pool
-
-import (
-	"crypto/ecdsa"
-	"crypto/elliptic"
-	"crypto/rand"
-
-	"github.com/nspcc-dev/neofs-sdk-go/client"
-	"github.com/nspcc-dev/neofs-sdk-go/session"
-	"github.com/nspcc-dev/neofs-sdk-go/user"
-)
-
-func ExampleNew_easiestWay() {
```

Comment:
Can we avoid moving them?

Reply:
You're mixing real changes with code movements and this makes the result hard to digest. If you're to split these changes into different commits --- I'm OK with that. If you're to just leave old tests the way they are (in separate files) --- even better.

Reply:
Ok, returned the original file and make two different commits, one - update existing code, the second - move the documentation